src/types/index.ts

```diff
@@ -143,4 +143,4 @@ export type CalloutIcon =
 
 export type CustomTransformer = (
   block: ListBlockChildrenResponseResult
-) => string | Promise<string>;
+) => string | false | Promise<string | boolean>;
```

Comment:
It looks like that `Promise<string | false>` won't work because TS infers its type as `Promise<boolean>` when I try to only return `false` in async callback.

**Example:**
```typescript
n2m.setCustomTransformer("embed", async (block) => {
  return false;
});
```
TS Error:
> Type 'Promise<boolean>' is not assignable to type 'string | Promise<string | false>'

Reply:
That is because over here https://github.com/souvikinator/notion-to-md/blob/8c15e9418204649bc0692fb520d8680b9f4fb4c6/src/notion-to-md.ts#L270 
if the value is false then it's inferring type as `string|true` (since boolean can be true as well as false)

A simple fix is to just check the type of the returned value as string in the if statement in the above lines instead of checking true/false.